components/dashboard/src/dedicated-setup/SetupCompleteStep.tsx

```diff
@@ -42,7 +42,7 @@ export const SetupCompleteStep: FC<Props> = ({ onComplete }) => {
             <Subheading>Your teammates can now sign in to Gitpod using single sign-on (SSO).</Subheading>
 
             <div className="flex flex-row items-center space-x-2 mt-4">
-                <div className="flex flex-row items-center space-x-1 font-mono text-sm text-gray-500 dark:text-gray-600">
+                <div className="flex flex-row items-center space-x-1 font-mono text-sm text-pk-content-secondary">
```

Comment:
This is a drive-by. I noticed that in the installation setup, the contrast of this text with the background is too small.